### DISCUSSION TITLE
Raylib SVG Rendering

### DISCUSSION THREAD

**Discussion by CelestialWren** [odin-lang/Odin#3809] [Q&A]
Raylib has SVG support natively, however it needs to be enabled through a config.h file to use. As this cannot be used in Odin, how do I change this configuration flag? the only configuration flags I can find are for the individual window you are rendering.

**Comment by SuperDragonite2172** [comment]
I've run into this issue as well.  In my research trying to figure out how to get it working it looks like SVG support was removed from base Raylib (2 weeks ago as of writing this comment, see https://github.com/raysan5/raylib/commit/1effe921291f5a260ef6e2841f0ed19ccb0c3e7c) and is being moved to "raylib-extras" which I can't find.  Maybe the repo is private or it hasn't been created yet but it sounds like the fix to enable SVG support will end up not working with Raylib 5.1.

**Comment by clankill3r** [comment]
I found an example here:
https://github.com/raylib-extras/raylib_multi_window_experimental/blob/4b1b19cc3a942e4b13d6296a787e3a0b5ee083d3/examples/textures/textures_svg_loading.c#L4

I guess if you want SVG you have to download raylib and compile the binaries yourself.

**Reply by SuperDragonite2172** [reply]
Thanks for finding that.  I'm like going to roll my own after discussing on the forums, but good to know that raylib-extras exists as a repo.

**Reply by JulienLecoq** [reply]
Arg I've lost many hours to find how to do it with the previous version of raylib and they made it even more difficult than before? I guess I'll have to loose many hours again to find how to make my code works again after this update... I really hate how raylib is managed, not user friendly at all.
